### ISSUE TITLE
Parsing of old Dahu presentation file fails when file is corrupted

### ISSUE THREAD

**Issue by dufourau** [dahuapp/dahu#53]
Bug when attributes misspelled in dahu file.
Examples: metadata is incorrect, metaData is correct.

**Comment by barraq** [comment]
The current parser for Dahu Screencast is minimalistic and breaks when properties are missing or when properties are misspelled (e.g. metadata instead of metaData).

To fix this bug we have to approches:
1. reject Dahu file format that are malformed. For that we just need to check if properties exist, if they don't we return an empty model or an error.
2. attempt to patch malformed Dahu file (e.g. check for metadata, Metadata, metaData, METADATA, etc.). This will be fastidious but still possible, for instance by checking for property.toLowerCase.

On the short term I suggest the first option, and eventually improve the tolerance of the parser in future version of Dahu.

**Comment by barraq** [comment]
@dufourau could you please check if this bug is now resolved?

**Comment by dufourau** [comment]
Yes I will, I'm currently working on the new parsing function (including coordinates in objects).
